templates/core/index.html

```diff
@@ -287,7 +287,7 @@ <h1>Secure embedded devices</h1>
             ) | safe
             }}
           </div>
-          <p class="p-muted-heading u-sv3" style="color: #fff; line-height: 1.5rem;">whitepaper</p>
+          <p class="p-heding--4">Whitepaper</p>
```

Comment:
A minor typo "heding" to "heading"

```suggestion
          <p class="p-heading--4">Whitepaper</p>
```

Reply:
Good spot thanks!